linux/shim/service.go

```diff
@@ -211,6 +212,7 @@ func (s *Service) Resume(ctx context.Context, r *shimapi.ResumeRequest) (*google
 }
 
 func (s *Service) Exit(ctx context.Context, r *shimapi.ExitRequest) (*google_protobuf.Empty, error) {
+	unix.Unmount(filepath.Join(s.path, "rootfs"), 0)
```

Comment:
Although not sure there will always be exactly one mount at this mountpoint even on Linux.